debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnection.java

```diff
@@ -117,6 +123,19 @@ public void readSchema(Tables tables, String databaseCatalog, String schemaNameP
                                     .create()
                                 );
                     }
+                    // NUMBER columns without scale value have it set to -127 instead of null;
```

Comment:
This should be kept in sync with https://github.com/debezium/debezium-incubator/blob/master/debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleDdlParser.java#L229